packages/default-reporter/src/reporterForClient/reportProgress.ts

```diff
@@ -156,16 +172,41 @@ function getProgessStatsPushStreamByRequirer (progress$: most.Stream<ProgressLog
   return progessStatsPushStreamByRequirer
 }
 
+function getPackageImportMethodPushStream (packageImportMethod$: most.Stream<PackageImportMethodLog>) {
+  const packageImportMethodPushStream = new PushStream<string>()
```

Comment:
I don't understand why a new pushstream is needed.

You can reuse the existing stream and you really need just one event. So like

```ts
packageImportMethod$
    .first()
    .map((log: PackageImportMethodLog) => {
      switch (log.method) {
        case 'copy':
          return 'copied'
        case 'clone':
          return 'cloned'
        default:
          return log.method
      }
    })
```

Reply:
It could be two events in auto mode where it first tries hardlink and fails, then copy